### ISSUE TITLE
Errors upon Correct Answer Reveal

### ISSUE THREAD

**Issue by a-pir1** [maybepiffpuffas/kahoot-flooder#1]
So, bot works pretty dam good, but I have been experiencing an issue since I have been using it.  I have been seeing that the console sends out errors once the correct answer is shown on the board.  It is not a big inconvenience, but it is just noticeable.

![image](https://user-images.githubusercontent.com/35549787/100762906-856b3600-33c2-11eb-913f-3cab5b351855.png)

**Comment by maybepiffpuffas** [comment]
Yes, I saw it and I'm trying to find out what causes that error, but I still can't find it. It might be because of an error in the library that I'm using.

**Comment by a-pir1** [comment]
If you ever consider open-sourcing this project or publishing the problematic code anywhere, let me know.  I am always down to help where I can, but from the looks of this error specifically, it is absolutely a library issue since Kahoot went through a lot of updates over the past 3-4 months.  It might not be a bad idea to suppress those errors for now if you can since they really don't help and hiding them won't hurt.
